doit/dependency.py

```diff
@@ -502,6 +501,12 @@ def get_value(self, task_id, key_name):
             raise Exception(msg % (task_id, key_name))
         return values[key_name]
 
+    def get_results(self, task_name):
+        """get all saved results from a task
```

Comment:
Each task has only 1 `result`, so it doesnt make sense to say "get all". And the function should be called `get_result()`.